acceptance/reproducibility_test.go

```diff
@@ -164,6 +164,12 @@ func compare(t *testing.T, img1, img2 string) {
 	cfg2, err := v1img2.ConfigFile()
 	h.AssertNil(t, err)
 
+	// images that were created locally may have `DockerVersion` equal to "dev" and missing `Config.Image`
+	cfg1.DockerVersion = ""
+	cfg2.DockerVersion = ""
```

Comment:
I tried explicitly setting this about here: https://github.com/buildpacks/imgutil/blob/67824e989715b0bdd3793f4df3d2e55d643052d8/local/save_file.go#L210 but it seems to have no effect.